src/skmultiflow/trees/nodes/sst_active_learning_node_adaptive.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+from skmultiflow.trees.nodes import SSTActiveLearningNode
+from skmultiflow.utils import get_dimensions
+
+
+class SSTActiveLearningNodeAdaptive(SSTActiveLearningNode):
```

Comment:
Please add a short description.

Reply:
Documentation updated in commit [5b724c9](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5b724c98a7bcc2d270f1034185b2978201235576).

Reply:
Documentation updated in commit [5b724c9](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5b724c98a7bcc2d270f1034185b2978201235576).